src/components/Explore/utils.ts

```diff
@@ -386,48 +382,47 @@ export const exploreMetricData: {
     ],
   },
 
-  // TODO(8.2) - update with real content:
   [ExploreMetric.ADMISSIONS_PER_100K]: {
     title: 'Hospitalizations per 100k',
     name: 'Hospitalizations per 100k',
-    chartId: 'positivity_rate',
+    chartId: 'admissions_per_100k', // TODO(8.2) (Chelsi) - what are these ids used for
```

Comment:
i _think_ these id's might have to do with the spark lines above the fold. ill poke around once both prs (this one and [the spark line pr](https://github.com/covid-projections/covid-projections/pull/5475)) are both merged into `can-82`

Reply:
I think maybe it's for sharing (when you share an explore chart we store the chartId so that when you follow the shared link you get the same chart again)

That said, sharing seems to be working even though these IDs don't match the `getMetricByChartId()` IDs used [above](https://github.com/covid-projections/covid-projections/pull/5476/files#diff-98bf8ba7ddd452a4cfd4b539f787207ddee93af012d0964ef5de6c783b60a09aL134). 

i.e. I'd think this should be `weeklyCovidAdmissionsPer100k` not `admissions_per_100k`, but somehow it still seems to be working.  I'm a bit confused, but I think I'd go ahead and make them match regardless?

Reply:
yeah im not totally sure // also confused. will investigate 🕵🏻‍♀️